src/hooks/spotify/useSpotifyAPI.tsx

```diff
@@ -37,11 +37,13 @@ function useSpotifyAPI(
     resetState();
     setLoading(true);
     try {
-      const result = await axios.get<CurrentPlaybackResponse>(
-        `${BASE_URL}${url}`,
-        buildHeaders(spotifyAuth),
-      );
-      setResponse(result.data);
+      if (httpMethod === 'GET') {
+        const result = await spotifyGETData(url, spotifyAuth);
```

Comment:
Seems better to just set the response in here so we can make a better determination of the status. Will make an issue for this.

Reply:
Foregoing this until we need it. Easy enough refactor